src/editors/object.js

```diff
@@ -1109,6 +1109,7 @@ export class ObjectEditor extends AbstractEditor {
 
     Object.keys(this.editors).forEach(i => {
       if (this.editors[i].isActive()) {
+        this.editors[i].refreshValue()
```

Comment:
For some reason after this change was introduced, the disable_array_delete_all_rows/disable_array_delete_last_row options are no longer working as expected. When the editor loads a schema with empty initial data for an array, the "delete all row" and "delete last row" buttons are always visible. Reverting this change seems to fix the issue. Additionally, adding and then removing a row also causes the buttons to disappear.

Reply:
Could you create an issue with an example.

Reply:
Sure: https://github.com/json-editor/json-editor/issues/1396